src/quo2/components/list_items/preview_list.cljs

```diff
@@ -109,40 +134,40 @@
 (defn border-type
   [type]
   (case type
-    (:account :collectible :photo) :rounded
-    :circular))
+    (:accounts :collectibles) :squared
+    :rounded))
 
-(defn- preview-list-internal
+(defn- view-internal
   "[preview-list opts items]
    opts
-   {:type          :user/:community/:account/:token/:collectible/:dapp
-    :size          32/24/16
-    :list-size     override items count in overflow label (optional)
-    :transparent?  overflow-label transparent?}
+   {:type          :user/:communities/:accounts/:tokens/:collectibles/:dapps/:network
+    :size          32/24/20/16/14
+    :number        number of items in the list (optional)
+    :blur?         overflow-label blur?}
    items           preview list items (only 4 items is required for preview)
   "
-  [{:keys [type size list-size transparent? theme more-than-99-label]} items]
+  [{:keys [type size number blur? theme more-than-99-label]} items]
```

Comment:
`number` is safe in cljs?

Reply:
I think it's safe as we don't have any number macro in Clojure. 

The closest is [`number?`](https://clojuredocs.org/clojure.core/number_q).

I used `number` to align with Figma. If we feel it's unsafe, we can change it 😄👍 .